web/src/views/HomeView/StatsBar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-container
+    class="grey darken-3 pa-0"
+    fluid
+  >
+    <v-row class="py-6">
+      <template v-for="stat in stats">
+        <v-col
+          :key="stat.name"
+          class="py-0 flex-grow-1"
+          md="2"
+          sm="4"
+          cols="6"
+        >
+          <SingleStat
+            :name="stat.name"
+            :value="stat.value"
+            :description="stat.description"
+          />
+        </v-col>
+        <!-- TODO dividers destroy the grid system breakpoints
+        <v-divider
+          :key="stat.name + '-divider'"
+          vertical
+          class="grey"
+        />
+        -->
+      </template>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import filesize from 'filesize';
+import SingleStat from '@/views/HomeView/SingleStat.vue';
+
+export default {
+  name: 'StatsBar',
+  components: { SingleStat },
+  computed: {
+    stats() {
+      return [
+        { name: 'dandisets', value: this.$store.state.stats.drafts.toString(), description: 'Draft dandisets, that is. Publishing is coming soon™' },
+        { name: 'users', value: this.$store.state.stats.users.toString() },
+        { name: 'species', value: this.$store.state.stats.species.toString() },
+        { name: 'subjects', value: this.$store.state.stats.subjects.toString() },
+        { name: 'cells', value: this.$store.state.stats.cells.toString() },
+        { name: 'total data size', value: filesize(this.$store.state.stats.size.toString(), { round: 0 }) },
```

Comment:
Using `mapState` here would help, instead of needing to specify `this.$store.state.stats` before each property

Reply:
Ah, know I see what you and Brian are getting at. I thought mapState could act as a macro or something. This is what I came up with:
```
  computed: mapState({
    stats: (state) => [
      { name: 'dandisets', value: state.stats.drafts.toString(), description: 'Draft dandisets, that is. Publishing is coming soon™' },
      { name: 'users', value: state.stats.users.toString() },
      { name: 'species', value: state.stats.species.toString() },
      { name: 'subjects', value: state.stats.subjects.toString() },
      { name: 'cells', value: state.stats.cells.toString() },
      { name: 'total data size', value: filesize(state.stats.size.toString(), { round: 0 }) },
    ],
  }),
```